### ISSUE TITLE
Poppy upgrade: Update schemas in folio-graphql

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/folio-graphql#162]
See https://github.com/sul-dlss/folio-graphql/?tab=readme-ov-file#updating-the-folio-schemas

... and try to patch over any backwards incompatibilities to make a seamless release easier.

... and validate that sul-requests + mylibrary graphql queries still work.

**Comment by hudajkhan** [comment]
Notes: 
Folio GraphQL = SUL's own fork of FOLIO GraphQL.  We need to synchronize schema updates. 
Ensure we can preserve old keys and new keys to have a seamless release process.
SUL requests and my library two major consumers of Folio GraphQL.

Two deployed environments for this repo: sul-folio-graphql-test - points at FOLIO test (poppy).  sul-folio-graphql-prod which points at FOLIO prod (not yet prod).

**Comment by marlo-longley** [comment]
Notes on this schema change:

- I reviewed the schema changes for required or non-optional new/modified fields and there are none used downstream in graphql queries.

- This is not a breaking change, but potentially a future ticket. One of the biggest changes this update introduced is a new type, HoldingsRecords, that we are not using. Instead we use an array of the existing type HoldingsRecord[] for example see https://github.com/sul-dlss/folio-graphql/blob/89353465bdca08cbb1af948b9561c1c9ceead2df/src/folio/holdings-api.ts#L17. We could update to use the new type HoldingsRecords for collections of HoldingsRecord but our code still works as is.

- RequestPolicyAllowedServicePoints could be useful to us in the future

- `fulfillmentPreference: RequestFulfillmentPreference!` -- we don't call this field to folio-graphql in mylibrary or sul-requests (in sul-requests we send this field to okapi).

- `mylibrary` makes 3 graphql queries which we've named LoanPolicies, ServicePoints, and PatronInfo. I made all these queries to an endpoint with the new schema and all worked as expected.

- `sul-requests` makes 3 graphql queries which we've named ServicePoints, InstanceByHrid, and CircCheckByBarcode. I made all these queries to an endpoint with the new schema and all worked as expected.
